lib/tracing/index.js

```diff
@@ -14,7 +14,7 @@ function _getSampler() {
   function _ignoreSpecifiedPaths(spanName, spanKind, attributes) {
     if (!cds.env.requires.telemetry.instrumentations?.http) return false
 
-    const { ignoreIncomingPaths } = cds.env.requires.telemetry.instrumentations.http
+    const { ignoreIncomingPaths } = cds.env.requires.telemetry.instrumentations.http.config
```

Comment:
```suggestion
    const { ignoreIncomingPaths } = cds.env.requires.telemetry.instrumentations.http.config || {}
```
protects against missing property `config`